source/reference/method/MongoDBClient-bulkWrite.txt

```diff
@@ -0,0 +1,100 @@
+============================
+MongoDB\\Client::bulkWrite()
+============================
+
+.. meta::
+   :description: Perform write operations across multiple namespaces by using the MongoDB PHP Library.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. versionadded:: 2.1
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\Client::bulkWrite()
+
+   Executes multiple write operations across multiple namespaces.
+
+   .. code-block:: php
+
+      function bulkWrite(
+          BulkWriteCommand|ClientBulkWrite $bulk,
+          array $options = []
+      ): MongoDB\BulkWriteCommandResult
+
+Parameters
+----------
+
+``$bulk`` : :phpclass:`MongoDB\ClientBulkWrite` or ``BulkWriteCommand``
```

Comment:
`MongoDB\Driver\BulkWriteCommand` can link to https://www.php.net/manual/en/class.mongodb-driver-bulkwritecommand.php